buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -224,7 +224,7 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
 
                     if (aComputer.isOffline()) {
                         echo "Node: ${JENKINS_URL}${aComputer.getUrl()} - Status: offline - Cause: ${aComputer.getOfflineCause().toString()}"
-                        offlineNodes.add(aComputer.getDisplayName())
+                        offlineNodes.add("<${JENKINS_URL}${aComputer.getUrl()}|${aComputer.getDisplayName()}>")
```

Comment:
`<a href="${JENKINS_URL}${aComputer.getUrl()}">${aComputer.getDisplayName()}</a>`
https://www.w3schools.com/html/html_links.asp

Reply:
That's not how Slack links work though. See L233 for example.

Reply:
https://api.slack.com/docs/message-formatting#linking_to_urls